src/main/java/it/gov/pagopa/payhub/activities/dao/TaxonomyDao.java

```diff
@@ -0,0 +1,7 @@
+package it.gov.pagopa.payhub.activities.dao;
+
+public interface TaxonomyDao {
```

Comment:
is this used?

Reply:
yes, for checking if the category of second beneficiary is in the archive of Taxonomy